spec/models/vacancy_spec.rb

```diff
@@ -29,4 +29,20 @@
     vacancy.valid?
     expect(vacancy.company_url).to eq("http://opensanca.com.br")
   end
+
+  context '.search' do
```

Comment:
To test a method, use `describe .search do`

And use 2 contexts to isolate the tests:

`context 'with a parameter'` and another
`context 'without a parameter'`